.idea/box-ios-sdk.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these .idea files from IntelliJ?  Do we want to commit them for others to use?

Reply:
I don't. I've already removed them in the next commit.